src/main/java/beans/menus/ErrorBean.java

```diff
@@ -0,0 +1,25 @@
+package beans.menus;
```

Comment:
just curious why you put this in the menu directory, seems like it belons in a formentry package or something